### ISSUE TITLE
Feature Request: Reload banner/logo from lutris.net

### ISSUE THREAD

**Issue by Mershl** [lutris/lutris#3126]
Creating and providing a known identifier will automagically download and set the banner and logo from lutris.net.

Feature Request: Allow retrigger of this download for when banner/logo on upstream changed.

**Comment by strycore** [comment]
this is an issue with the website, moving to the correct repo

**Comment by Mershl** [comment]
To clarify. The feature request is suggesting the Lutris client having the option to reload the current banner from the website.

**Comment by strycore** [comment]
yes, it should already do that when the banner updates. But some changes in how we host banners have been made and now it does not correctly update because the old one gets cached forever.
